_posts/2023-08-26-mev-cui-bono.md

```diff
@@ -0,0 +1,169 @@
+
+In turn, CowSwap and other protocols that offer this kind of services will have to get their bundles into a block. Unless they do the block building themselves (and often they don't have enough orderflow to), they will have to rely on a third party block-builder, that will ask for its cut and will bring the end cost to the user even higher.
+
+So, all in all, what PBS is solving isn't really much of a UX problem: Indeed, the UX is probably getting economically worse for the end user in many circumstances - especially for users moving little money, which are less likely to be pray of MEV bots. PBS was intended as a solution to a centralization problem, fragmenting and disciplining a monolithic infrastructure, at the proposer layer. 
+
+But what happens to rest of the value chain? And how does the rest of the value chain affect proposers?
+
+## Here comes centralization
+
+In a [recent tweet](https://twitter.com/specialmech/status/1691178038640492544), Special Mechanism Group highlights how integrated builder-searchers are on the rise. An integrated builder-searcher is nothing more than a searcher that bids directly for blockspace.
+
+This should not be conceptually surprising, and it is indeed a very old phenomenon: Depending on my own capabilities and the price setting by third parties, I can *verticalize* my operations. If sufficiently effective, I will now pay the previously outsourced goods at lower costs, taking out fees an intermediary could take. Over time, through learning, I might be able to further reduce my costs and increase my margin.
+
+What is interesting though, there might be an additional rationale at work: Through integration I, as a searcher, might erect barriers of entries that make it harder for a new competitor to enter the search space. Such moves per se are not specific to PBS or crypto for that matter but are standard topics for (controversial) discussions in markets.
+
+### Integrating proposers
+
+Indeed, at 20[ ] we're very prone to consider builders and searchers as conceptualy isomorphic: A profitable searcher with sufficient capital will inevitably venture into block building to be able to increase margins. While the logic of integration for costs reasons does not readily apply to proposer integration - remember, a business intentionally designed to be relatively dumb - the logic of integration for reducing competition possibly does.
```

Comment:
I don't really get what you're saying in the last period

Reply:
It means that you don't become a proposer to better your costs, but because this gives you the possibility of pushing other searchers out of the market.

Reply:
Okay, this doesn't flow super smoothly but I don't have a better wording to propose, so it's fine for me